app/components/lead_providers/your_schools/table.html.erb

```diff
@@ -1,16 +1,18 @@
-<table data-test="schools-table" class="govuk-table lead-provider-your-schools-table">
-  <thead>
-    <tr class="govuk-table__row">
-      <th scope="col" class="govuk-table__header">School</th>
-      <th scope="col" class="govuk-table__header">URN</th>
-      <th scope="col" class="govuk-table__header">Delivery partner</th>
-      <th scope="col" class="govuk-table__header govuk-table__header--numeric">ECTs</th>
-      <th scope="col" class="govuk-table__header govuk-table__header--numeric">Mentors</th>
-    </tr>
-  </thead>
-  <tbody class="govuk-table__body">
-    <%= render LeadProviders::YourSchools::TableRow.with_collection(partnerships) %>
-  </tbody>
-</table>
+<div class="nhsuk-table-container">
+  <table data-test="schools-table" class="govuk-table nhsuk-table-responsive lead-provider-your-schools-table">
+    <thead  class="nhsuk-table__head">
+      <tr class="govuk-table__row">
```

Comment:
Does this want `nhsuk-table__row`?

Reply:
Not required in header, from the looks of things: https://service-manual.nhs.uk/design-system/components/table